domain/src/main/java/com/thoughtworks/go/domain/Stage.java

```diff
@@ -35,6 +35,7 @@
     private String name;
     private JobInstances jobInstances = new JobInstances();
     private String approvedBy;
+    private String cancelledBy;
```

Comment:
Just a question - `String`? Can this be `Username`?
We might require to compare usernames (to know who canceled). 
Instead of operating on primitives, it is better to wrap it in an object.